build.sbt

```diff
@@ -47,6 +47,61 @@ val dep = {
   )
 }
 
+val verifyDeps = Seq(
```

Comment:
There are a couple jars that are only used for testing, but their signature is not checked, should it be also checked?
```
"com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % "it,test"
"com.miguno.akka" %% "akka-mock-scheduler" % "0.5.1" % "it,test"
```

Reply:
They don't go into the final package anyway, so I'm not sure if it's worth doing that (it would probably require some changes in the plugin code)